include/Game/Animation.h

```diff
@@ -6,23 +6,26 @@
 
 #include "Util/Time.hpp"
 
-class Animation {
+class Animation final {
 public:
     Animation(const glm::vec2& animationPosition);
 
-    virtual ~Animation() = default;
-
-    virtual void MoveByTime(
+    void MoveByTime(
         const unsigned long duringTimeMs,
         const glm::vec2&    destination,
         const uint16_t      direction
     );
-    virtual void
+    void
     MoveByTime(const unsigned long duringTimeMs, const glm::vec2& destination);
-
-protected:
-    void UpdateAnimation(const bool& isDirection = false);
-
+    void      UpdateAnimation(const bool& isDirection = false);
```

Comment:
```suggestion
    void      UpdateAnimation(bool isDirection = false);
```
It is Huge cost that to use reference to transfer param